cypress/utils/channelsUtils.js

```diff
@@ -1,52 +1,50 @@
 import Channels from "../apiRequests/Channels";
 
-class ChannelsUtils {
-  channels = new Channels();
-  createdChannel;
+const channels = new Channels();
```

Comment:
as discussed in previous PRs - avoid module level variables.

Reply:
Yes, I will fix it in resolving conflicts from previous PR